src/lib/mutations/createContent/createContent.hook.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  useMutation,
+  UseMutationOptions,
+  UseMutationResult
+} from '@tanstack/react-query'
+
+import {
+  createContent,
+  CreateContentData,
+  CreateContentVariables
+} from './createContent'
+
+export function useCreateContent(
+  options?: Omit<
+    UseMutationOptions<
+      CreateContentData,
+      unknown,
+      CreateContentVariables,
+      unknown
+    >,
+    'mutationFn'
+  >
+): UseMutationResult<CreateContentData, unknown, CreateContentVariables> {
+  return useMutation(createContent, options)
```

Comment:
This optimistically saves the content right? what happens if the client errors out?

Reply:
no optimistic saving going on here.